library/rpi_ws281x/rpi_ws281x.py

```diff
@@ -141,7 +131,7 @@ def show(self):
     def setPixelColor(self, n, color):
```

Comment:
These methods become pretty useless, but should be kept for a while to avoid having any breaking changes